### PR TITLE
175 update dashboard

### PR DESCRIPTION
**Pull request by smebrahimi**
This issue #175

### CODE REVIEW

**Comment by atharva3010** [comment]
image
I think there should not be a blue background behind the preview image here

**Comment by paulchrisluke** [comment]
This review done through: https://deploy-preview-210--vw-email-cms.netlify.app/

image
We are not using this temporary home/dashboard page anymore. We are replacing this one with our new dashboard page (below). 
image

# Log In 
When you log in, this is the page that needs to be displayed. 
image

# Bottom Navigation 
image
When you click "home", users should be redirected to the new dashboard

# Settings Back Button 
gif
When you press "back" from the new dashboard page, it should not go back to the old "welcome to blue delta" home page, it should go to Dashboard

File: src/components/TodayEmail.vue [hunk 1]
```diff
@@ -25,11 +21,34 @@
...
+      .then(() => {
+        let article, i;
+        this.update_selected_jumpstart(this.get_jumpstarts[0]);
+        for (i = 0; i < 5; i++) {
+          article = this.get_selected_jumpstart.articles[i];
+          if (article) this.update_selected_articles(article);
+          else break;
+        }
+        this.update_preview_link('');
+        this.debounced_preview();
```
Comment (atharva3010) [inline src/components/TodayEmail.vue:42]: I think we don't need to use debounced_preview here, normal preview would be better.

**Comment by smebrahimi** [comment]
> image
> I think there should not be a blue background behind the preview image here, also the countdown timer's data is weird.

@bamboochow ^ Background change?

**Comment by paulchrisluke** [comment]
> > image
> > I think there should not be a blue background behind the preview image here, also the countdown timer's data is weird.
> 
> @bamboochow ^ Background change?

Will put this in a separate backlog issue

File: src/components/TodayEmail.vue [hunk 2]
```diff
@@ -4,16 +4,12 @@
...
-        height="373"
-        width="264"
-      />
+      <EmailPreview />
     </div>
     <div>
       <h2 class="tg-h2-mobile text-primary">Daily Email will automatically send in</h2>
       <div class="counter tg-h2-mobile my-2 text-center">
-        <BaseCounter date="Jan 5, 2021 15:37:25" />
+        <BaseCounter date="May 15, 2020 15:37:25" />
```
Comment (atharva3010) [inline src/components/TodayEmail.vue:12]: Can we use the date from the first jumpstart?

File: src/components/TodayEmail.vue [hunk 3]
```diff
@@ -25,11 +21,33 @@
...
+    this.fetch_jumpstarts()
+      .then(() => {
+        let article, i;
+        this.update_selected_jumpstart(this.get_jumpstarts[0]);
+        for (i = 0; i < 5; i++) {
+          article = this.get_selected_jumpstart.articles[i];
+          if (article) this.update_selected_articles(article);
+          else break;
+        }
+        this.update_preview_link('');
```
Comment (atharva3010) [inline src/components/TodayEmail.vue:41]: When we pass `''` as payload, it resets the preview link. Call the preview link method without any payload.
Like this `this.update_preview_link()`

File: src/components/TodayEmail.vue (hunk 1 above)
Comment (smebrahimi) [reply src/components/TodayEmail.vue:41]: Without this.debounced_preview(); preview not work

**Pull request merged**